daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -1116,6 +1202,38 @@ class DecodeV1Spec
         ),
         signatoryInterface -> ESignatoryInterface(ifaceId = scalaIfaceTyConName, body = EUnit),
         observerInterface -> EObserverInterface(ifaceId = scalaIfaceTyConName, body = EUnit),
+        toInterface -> EToInterface(
```

Comment:
we were missing a few, thanks for reminding me to check @remyhaemmerle-da

Reply:
:bow: